src/plugins/external/cookie.ts

```diff
@@ -0,0 +1,11 @@
+import fastifyCookie from '@fastify/cookie'
+import fp from 'fastify-plugin'
+
+/**
+ * This plugins enables the use of cookies.
+ *
+ * @see {@link https://github.com/fastify/fastify-cookie}
+ */
+export default fp(fastifyCookie, {
+  name: 'cookies'
+})
```

Comment:
Is this file needed at all? Can we do it together with `session`?

Reply:
We can